### ISSUE TITLE
CJ4 fuel loss

### ISSUE THREAD

**Issue by Sotiris1955** [Working-Title-MSFS-Mods/fspackages#1437]
After the SU7 and the last working CJ4 mod v.0.12.9, and while taking off,there is a dramatically loss of fuel.

**Comment by dga711** [comment]
Make sure fuel dump isn't on by accident. Check your control bindings if anything is bound to that, you didnt know of.

**Comment by Sotiris1955** [comment]
Thank’s,I will check it out.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by Raikeninc** [comment]
I can confirm this issue. At takeoff and starting phase of climb fuel is is burning very fast by "portions" of 50-100 kg in few seconds. And that is happening in about every half of a minute. On cruise everything is fine.
It is happens in every flight (got 5 for now). 
Maybe it is a FSA Client issue, didn't have time to check without it.

**Comment by Sotiris1955** [comment]
Thank you very much !!!
Looking forward to be fixed soon.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

________________________________

**Comment by OliveBeast20441** [comment]
I wish to confirm this also. I reverted back to version 0.12.9 and this bug is still present. I do not have any bindings on the keyboard or joystick associated with fuel dumping.  The fuel loss occurs after takeoff for about 30-60 seconds, then comes back to normal, as stated by Raikeninc, above. 

Other aircraft do not exhibit this behaviour; only the CJ4 and the control bindings are not changed when I fly other aircraft. Also, it sporadic, happening whether starting cold & dark or spawned on the runway.

**Comment by OliveBeast20441** [comment]
I just found a work around to stop this excessive fuel consumption - change the view from cockpit to external, then back to cockpit and the fuel flow returns to normal.

I have no clue why this works, but it does. I hope the developers can figure this one out!  :-)

**Comment by KaitlynnUK** [comment]
This was a flight from RJTK with a set altitude of 20k.

At just over 16k feet my fuel went from about 1300lbs in each tank to around 850lbs in each tank in about 2 seconds. I used the Toolbar fuel sider to set it back to around 1300lbs.

However, about a minute later the fuel gauges in both tanks changed drastically again within about 2 seconds, but this time leaving me with -94lbs in the left tank and 11370lbs in the right! 

![image](https://user-images.githubusercontent.com/40742801/154802553-ba882865-2a49-4fe6-9cfa-e258be640df8.png)

**Comment by KaitlynnUK** [comment]
> Kaitlynn,
> 
> If you see this happening again, I found that by switching the view from cockpit to external, then back to cockpit stops the excessive fuel consumption. I have no idea why this works, but it does.
> 
> Try it the next time you see this happening.
> 
> Best regards,
> 
> Mark

It happens within 2 seconds, my reactions are not that fast sadly :)

**Comment by Persojet** [comment]
This is major problem now with the SU9 and 12.13 for me. Happpens on every flight. Sometimes it drains almost all the fuel at take off and sometimes just so I can get to my destination.

**Comment by LeCoyote** [comment]
Agreed. I just burned through 800kg of fuel to get a 6t CJ4 up to FL300. It's a major issue for sure!

**Comment by SkyDog** [comment]
Had the same Problem today with v0.12.14. Did 10 flights during the last days without a problem. First occurence today. I only did two things different compared with previous flights:

Video of the occurence: https://youtu.be/qHUaEIyh1_Q

1.) Being the second flight today I didn't restart the sim in between the flights. You can still see the flight log of the first flight on FMS with earlier T/O and LDG time. Note that it says FUEL USED 12KG ... I checked with the video of the previous flight - there its clearly seen that it says FUEL USED 795KG. Value changed after switching off BATTERY and switching on again for the second flight - T/O and LDG time however remains.

2.) Used the W/B (Fuel) function "in-game" today for the first time (as seen in video). On earlier flights I did the fueling in the WORLD menu before jumping into the cockpit.

hope this helps ...
